python-lib/plugin_config_loading.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""Module with utility functions for loading, resolving and validating plugin parameters"""
+
+import logging
```

Comment:
Several imports are not used, can you remove those?
Pro-tip: Install flake8 on VScode

Reply:
Still unused imports